src/main/java/org/opentripplanner/standalone/CommandLineParameters.java

```diff
@@ -140,25 +140,27 @@
             description = "Allow unauthenticated access to sensitive API resources, e.g. /routers")
     public boolean insecure = false;
 
-    @Parameter(names = { "--script" }, description = "run the specified OTP script (groovy, python)")
+    @Parameter(names = {"--script"}, description = "run the specified OTP script (groovy, python)")
     public File scriptFile = null;
 
-    @Parameter(names = { "--enableScriptingWebService" }, description = "enable scripting through a web-service (Warning! Very unsafe for public facing servers)")
+    @Parameter(names = {"--enableScriptingWebService"}, description = "enable scripting through a web-service (Warning! Very unsafe for public facing servers)")
     boolean enableScriptingWebService = false;
 
-    @Parameter(names = { "--sharedVehiclesApi" }, description = "API for fetching info about renable vehicles")
+    @Parameter(names = {"--sharedVehiclesApi"}, description = "API for fetching info about renable vehicles")
     public String sharedVehiclesApi = null;
 
-    @Parameter(names ={"--trafficAppi"},description = "API for trafic udate")
-    public  String traffic = null;
+    @Parameter(names = {"--trafficAppi"}, description = "API for traffic update")
```

Comment:
This is not something associated with recent changes, but perhaps we could also fix the typo in the parameter name

Reply:
Yup, I want to fix this typo, but after migration to kubernetes.